Start filling out the readme

Robdel12 opened the pull request:
Will close #9 & #6 
- [x] TODO: needs rebase

Robdel12 committed:
Start filling out the readme

cowboyd commented on README.md:
```diff
@@ -1,2 +1,37 @@
+A project champion is a person who is the main manager of an open
+source repo. This person will have the most context of the project and
+documents the roadmap of the project. This does not mean they do all
+the work.
```
I would explain why we have the need to have a project champion first. I.e. "unowned projects rot and die of gangrene" and so we need to make sure that there is someone who owns the project and is its advocate both internally and externally. etc... etc..

Robdel12 replied:
I like this very much! I'll work it in.

cowboyd commented on README.md:
```diff
@@ -1,2 +1,37 @@
+Being a project champion means managing the code is only half of the
+battle. You're managing a small community that uses your project with
+it. You also fill a support role for people who need help using the
+project. It's very important to foster this community!
```
It would be awesome if there were a way that anybody could determine at a glance who was the champion of a project. Don't know how this would work though.

Robdel12 committed:
Added more content to readme &...

Documented the weekly email and meeting process, who is a project
champion and of what project.

Robdel12 committed:
Move weekly emais to monthly folders

Robdel12 committed:
First stab at "why is OSS imporant to the frontside"

Robdel12 committed:
I can't spell. So I fixed some spelling errors

Robdel12 committed:
Small improvements for readability

Robdel12 commented on README.md:
```diff
@@ -1,2 +1,107 @@
+
+Being a project champion means managing the code is only half of the
+battle. You're managing a small community that uses your project with
+it. You also fill a support role for people who need help using the
```
> You're managing a small community that uses your project with it.

"You're managing a small community that uses your project."

cowboyd committed:
a few edits to the "why" open source (#15)

Add more rhetorical emphasis on the material benefits.

Robdel12 committed:
Small update to readme

Robdel12 committed:
Move emails to the right dirs
